src/core/ext/filters/client_channel/subchannel_pool_interface.cc

```diff
@@ -94,4 +94,12 @@ SubchannelPoolInterface::GetSubchannelPoolFromChannelArgs(
   return static_cast<SubchannelPoolInterface*>(arg->value.pointer.p);
 }
 
+grpc_channel_args* SubchannelPoolInterface::RemoveSubchannelPoolArg(
```

Comment:
If we don't ref the subchannel pool when copying the arg, we don't need this helper method.